### DISCUSSION TITLE
AES EAX encryption/decryption - Documentation

### DISCUSSION THREAD

**Discussion by theraaz** [openpgpjs/openpgpjs#1540] [Q&A] [answered]
Hi, I'm trying to update my lib from 4.10.1 to the latest. I tried to go through the documentation but couldn't find the replacement of EAX decryption. The tests folder directly imports the EAX module to test however on production env, the EAX cannot be imported like that. Here is what I was doing before:

```js
const eax = await window.openpgp.crypto.eax(cipher, key);
let result = await eax.decrypt(encryptedData, nonce, header);
```
How can I decrypt the same using latest version 5.3?

**Comment by twiss** [comment, marked as answer]
Hey :wave: There is no replacement. OpenPGP.js is an OpenPGP library, not a low-level crypto library, and beyond that, EAX support (and AEAD in general) was and is experimental, and may change or go away at any point, and so should not be relied upon. I would recommend using a different library, or extracting the EAX code from OpenPGP.js.
